src/components/manage/Blocks/Accordion/editor.less

```diff
@@ -47,6 +47,14 @@
     padding-top: 1rem;
     padding-bottom: 0.1rem;
     margin: 0;
+
+    .ui.styled.accordion {
```

Comment:
@razvanMiu This is changing the panel title color in `.active` "Edit" mode.

Reply:
If its the requirement https://eea.github.io/volto-kitkat-frontend/?path=/story/components-accordion--default
Then it should go somewhere in design-system theme, imo.